hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/DiskBalancerStatusSubcommand.java

```diff
@@ -64,7 +64,7 @@ private String generateStatus(
 
     List<String> contentList = new ArrayList<>();
     contentList.add("Datanode");
-    contentList.add("VolumeDensity");
+    contentList.add("BalancedBytes");
```

Comment:
Let's move the BalancedBytes or BytesMoved before "EstBytesToMove", and show the MB value.